scripts/travis/publish.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * Publish the ddoc from the testing db to staging.
+ */
+const https = require('https'),
+      http = require('http'),
+      {
+        UPLOAD_URL,
+        TRAVIS_BUILD_NUMBER,
+        TRAVIS_TAG,
+        TRAVIS_BRANCH
+      } = process.env,
+      releaseName = TRAVIS_TAG || TRAVIS_BRANCH;
+
+const PouchDB = require('pouchdb-core');
+PouchDB.plugin(require('pouchdb-adapter-http'));
+
+if (!releaseName) {
```

Comment:
What else could this be?

Reply:
Oh is this blocking PR builds? Maybe make this clearer, that whole PR build thing is hugely confusing and so we should be cautious and clear when we dance around it

Reply:
Ok, I've added to the comment. Does it make it clearer?